docs/cip36-voting-registration-example.md

```diff
@@ -24,48 +24,48 @@ cardano-hw-cli address key-gen \
 --hw-signing-file stake.hwsfile
 ```
 
-Get slot number from `cardano-cli`, use slot number as `nonce` in governance voting registration command:
+Get slot number from `cardano-cli`, use slot number as `nonce` in CIP36 voting registration command:
 ```
 cardano-cli query tip --mainnet
 ```
 
-Get stake address from `cardano-cli`, use it as `reward-address` and `reward-address-signing-key` in governance voting registration command:
+Get stake address from `cardano-cli`, use it as `payment-address` and `payment-address-signing-key` in CIP36 voting registration command:
```

Comment:
Here we recommend using a stake address, which is no longer correct if I recall correctly.